src/main/java/service/ReaderImpl.java

```diff
@@ -0,0 +1,47 @@
+package service;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import model.Fruit;
+
+public class ReaderImpl implements Reader {
+    private static final String FILE_SEPARATOR = FileSystems.getDefault().getSeparator();
+    private static final String INPUT_FILE = "src" + FILE_SEPARATOR
+            + "main" + FILE_SEPARATOR + "resources" + FILE_SEPARATOR + "input.csv";
+
+    @Override
+    public String read() {
```

Comment:
Lets pass path to file as incoming parameter, so we can use ReaderImpl to read different files. And I think it's better to return list of lines List<String> instead of joining them

Reply:
I kept String for now 'cause it's a bit complicated to work with the List in the getFromCsvRow(String line) method